### PR TITLE
Performing smoke tests on all APIs on deploy

### PR DESCRIPTION
**Pull request by giorgiosironi**

### CODE REVIEW

**Comment by giorgiosironi** [comment]
I get an error locally, and we are having some similar problem on `/covers` in production. Let's see if we can make this green.

**Comment by nlisgo** [comment]
I will look at this tomorrow.

**Comment by giorgiosironi** [comment]
This has become green on `ci` without changing the code, so it may depend on the state of the db. Which is a good reason to merge it so that we can verify it on all envs.

Unrolling the loop now.

File: smoke_tests.sh [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+hostname=${1:-$local_hostname}
+
+echo "Homepage"
+[ $(curl --write-out %{http_code} --silent --output /dev/null "$hostname") == 200 ]
+
+echo "APIs"
+[ $(curl --write-out %{http_code} --silent --output /dev/null -H "Accept: application/vnd.elife.blog-article-list+json; version=1" "${hostname}/blog-articles") == 200 ]
+[ $(curl --write-out %{http_code} --silent --output /dev/null -H "Accept: application/vnd.elife.collection-list+json; version=1" "${hostname}/collections") == 200 ]
+[ $(curl --write-out %{http_code} --silent --output /dev/null -H "Accept: application/vnd.elife.cover-list+json; version=1" "${hostname}/covers") == 200 ]
+[ $(curl --write-out %{http_code} --silent --output /dev/null -H "Accept: application/vnd.elife.event-list+json; version=1" "$hostname"/"${hostname}/events") == 200 ]
```
Comment (nlisgo) [inline smoke_tests.sh]: You have repeated `$hostname`:

```
[ $(curl --write-out %{http_code} --silent --output /dev/null -H "Accept: application/vnd.elife.event-list+json; version=1" "$hostname"/"${hostname}/events") == 200 ]
```

Comment (giorgiosironi) [inline smoke_tests.sh]: sorry, too many amendments, correcting it now

**Pull request merged**